### PR TITLE
be able to add/remove/switch tracks

### PR DESCRIPTION
**Pull request by jensengar**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -237,9 +237,62 @@ MediaSession.prototype = extend(MediaSession.prototype, {
...
+    // Media control methods
     // ----------------------------------------------------------------
 
+    findTransceiverForTrack: function (track, failIfNotFound) {
+      if (failIfNotFound !== false) {
+        failIfNotFound = true;
+      }
+
+      var transceiver = null;
+      this.pc.pc.getTransceivers().forEach(function (tc) {
```
Comment (xdumaine) [inline index.js]: Can you use `find` here instead of looping the entire set?

File: index.js [hunk 2]
```diff
@@ -237,9 +237,62 @@ MediaSession.prototype = extend(MediaSession.prototype, {
...
+      }
+
+      return transceiver;
+    },
+
+    addTrack: function (track, dontReuse) {
+      var availableTransceiver = null;
+
+      // reuse available transceiver
+      this.pc.pc.getTransceivers().forEach(function (tc) {
```
Comment (xdumaine) [inline index.js]: Same here

**Review by xdumaine — COMMENTED** [review]
Should we have handling in here for browsers that don't have transceivers even when polyfilled? or Does adapter add them even in older versions of Chrome?

**Comment by jensengar** [comment]
It looks like adapter is always shimming chrome. according to mdn, chrome doesn't support transceivers natively yet.

**Pull request merged**